development/docker-compose.dev.yml

```diff
@@ -33,9 +25,15 @@ services:
       disable: true
     tty: true
   worker:
+    entrypoint:
+      - "sh"
+      - "-c"  # this is to evaluate the $NAUTOBOT_LOG_LEVEL from the env
+      - "watchmedo auto-restart --directory './' --pattern '*.py' --recursive -- nautobot-server celery worker -l $$NAUTOBOT_LOG_LEVEL --events"  ## $$ because of docker-compose
```

Comment:
I see now seeing this, probably the removal of the `watchmedo` in development/docker-compose.base.yml doesn't matter